src/components/link.tsx

```diff
@@ -72,7 +75,8 @@ export class Link extends React.Component<LinkProps, {}> {
         event.preventDefault();
 
         // Change the router state to trigger a refresh
-        const { routerStore, toState } = this.props;
+        const { routerStore, toState, onClick } = this.props;
+        if (onClick != null) onClick(event);
```

Comment:
If an external `onClick` handler is defined then this code will never be called (because `handleClick` will never be called). Try it.